.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
+
+**Docs**
+
+- [ ] Reference docs are published to istio.io. 
```

Comment:
would we allow istio wiki for alpha feature?

Reply:
I think that's fine.